cmds/exp/pox/pox.go

```diff
@@ -277,21 +295,46 @@ func poxRun(args []string) error {
 	return nil
 }
 
+func extraMounts() error {
+	if *extra == "" {
+		return nil
+	}
+	v("Extra: %q", *extra)
+	// We have to specify the extra directories and do the create here b/c it is a squashfs. Sorry.
+	for _, e := range strings.Split(*extra, ",") {
+		m := mp{flags: mount.MS_BIND, perm: 0755}
+		mp := strings.Split(e, ":")
```

Comment:
Go noob here, but when you use 'mp' here on the left of a :=, right below using 'mp' as the type mp (mountpoint), is a little confusing.